indexer/services/roundtable/src/tasks/update-affiliate-info.ts

```diff
@@ -36,6 +38,12 @@ export default async function runTask(): Promise<void> {
       ? persistentCacheEntry.value
       : defaultLastUpdateTime);
 
+    // Track how long ago the last update time (windowStartTime) in persistent cache was
+    stats.gauge(
+      `${config.SERVICE_NAME}.persistent_cache_${PersistentCacheKeys.AFFILIATE_INFO_UPDATE_TIME}_lag_seconds`,
+      DateTime.utc().diff(windowStartTime).as('seconds'),
+    );
```

Comment:
nit: actually since we can track this in the same dashboard let's do this
```suggestion
    stats.gauge(
      `${config.SERVICE_NAME}.persistent_cache_lag_seconds`,
      DateTime.utc().diff(windowStartTime).as('seconds'),
      { cache: PersistentCacheKeys.AFFILIATE_INFO_UPDATE_TIME.},
    );
```

Reply:
and also for the other task

Reply:
makes sense